origami/types/files.py

```diff
@@ -378,3 +378,30 @@ class ExistsResult(BaseModel):
     """The response payload for existence check requests"""
 
     exists: bool
+
+
+class FileVersion(BaseModel):
+    """A collection of file deltas were squashed to create this version"""
+
+    id: UUID
+    created_at: datetime
+    updated_at: datetime
+    deleted_at: Optional[datetime] = Field(
+        description="null if the latest unsaved version or not deleted"
+    )
+    created_by_id: Optional[UUID]
+
+    number: Optional[int] = Field(
```

Comment:
```suggestion
    version_number: Optional[int] = Field(
```

Reply:
This model comes from a Gate API response, so we might have to leave it this way for now.